run_check_material_profiles.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -eu
+
+. ./make_docker.sh
+
+run_in_docker python3 scripts/check_material_profiles.py || echo "Failed!"
+
+exit 0
```

Comment:
Since you've listed lxml as a dependency, you can remove this script and just use the XSD test.

Reply:
@bryanfoley Can I remove this script? It'd be ready for merging then in my opinion.

Reply:
@Ghostkeeper yes you can remove the script.